lib/modules/platform/gerrit/client.ts

```diff
@@ -149,18 +149,25 @@ class GerritClient {
   ): Promise<void> {
     await this.gerritHttp.postJson(
       `a/changes/${changeNumber}/revisions/current/review`,
-      { body: { labels: { [label]: value } } },
+      { body: { labels: { [label]: value }, notify: 'NONE' } },
     );
   }
 
-  async addReviewer(changeNumber: number, reviewer: string): Promise<void> {
-    await this.gerritHttp.postJson(`a/changes/${changeNumber}/reviewers`, {
-      body: { reviewer },
-    });
+  async addReviewers(changeNumber: number, reviewers: string[]): Promise<void> {
+    await this.gerritHttp.postJson(
+      `a/changes/${changeNumber}/revisions/current/review`,
```

Comment:
this breaks older Gerrit versions. since when this API exists? 😨

Reply:
I use Gerrit 3.4, of which this API works.

![image](https://github.com/user-attachments/assets/a2876560-0eca-4889-bc87-12ca5fdc8aeb)

What is the version of Gerrit you are using?

IIRC this endpoint is pretty old. Maybe the adding reviewers functionality isn't.

Reply:
@viceice, installed Gerrit 3.0 with `docker run --rm -p 8080:8080 gerritcodereview/gerrit:3.0.16` and its REST API docs still says it's supported:

![image](https://github.com/user-attachments/assets/6febe5fb-d765-4f2c-978b-633458b63740)

But I surely did not mean to break this functionality for anyone. And it is working here, I tested for real with Gerrit 3.4 several times.

Can you please watch your Renovate logs to see if it prints anything when it fails?

Reply:
ok, the readme says we support all 3.x versions, so please ensure compatibility if possible.

otherwise update readme and raise the minimum supported version.

Reply:
@viceice, maybe I did not make myself clear:

This API **is supported** by all Gerrit 3.x versions.

If you have noticed that it broke, it is unexpected, and I would like to debug it.

Reply:
This PR is ok, i don't use Gerrit. But we like to ensure to not break users if possible (even this platform is alpha grade).

Just make sure do add some extended descriptions to the PR for changed API and validate the mentioned supported versions are not broken. If it needs to break, then also update the readme to reflect it. also add a description why we need to break it.

Hopefully this is more clear now 🤗

Reply:
@viceice, understood.

Be sure I would have made it explicit in case of a breaking change.

I am positive this API change did not break any existing user of Renovate.